module/stylesheets/components/_chip.scss

```diff
@@ -38,6 +38,11 @@
 	    margin: 4px 4px 0px -8px;
 	    border-radius: 50px;
     }
+
+    .avatar{
+        height: 32px;
```

Comment:
Shall we use relative sizes. I believe this will broke if we change the size on the chip also this should nested with `.chip` class